### PR TITLE
feat: flexible order time selection ui changes

### PR DESCRIPTION
**Pull request by nada-deriv**
## Changes:

ui changes for flexible order time selection => dropdown in create/edit ad, new modal onclicking info icon

### Screenshots:

Please provide some screenshots of the change.
Screenshot 2023-08-24 at 9 50 38 AM
Screenshot 2023-08-24 at 9 51 04 AM

Screenshot 2023-08-24 at 10 02 41 AM

### CODE REVIEW

File: packages/p2p/src/components/my-ads/edit-ad-form.jsx [hunk 1]
```diff
@@ -149,6 +150,7 @@ const EditAdForm = () => {
                             max_transaction: max_order_amount_display,
                             min_transaction: min_order_amount_display,
                             offer_amount: amount_display,
+                            //order_completion_time: order_completion_time,
```
Comment (ameerul-deriv) [inline packages/p2p/src/components/my-ads/edit-ad-form.jsx:154]: could we add a comment above it to say to uncomment this once BE has been released for order_completion_time 👀

File: packages/p2p/src/components/modal-manager/modals/order-time-tooltip-modal/order-time-tooltip-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+
+const BlockUserModal = () => {
```
Comment (farhan-nurzi-deriv) [inline packages/p2p/src/components/modal-manager/modals/order-time-tooltip-modal/order-time-tooltip-modal.tsx]:
```suggestion
const OrderTimeTooltipModal = () => {
```

File: packages/p2p/src/components/modal-manager/modals/order-time-tooltip-modal/order-time-tooltip-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            <Modal.Footer>
+                <Button primary large onClick={hideModal}>
+                    <Localize i18n_default_text='Ok' />
+                </Button>
+            </Modal.Footer>
+        </Modal>
+    );
+};
+
+export default BlockUserModal;
```
Comment (farhan-nurzi-deriv) [inline packages/p2p/src/components/modal-manager/modals/order-time-tooltip-modal/order-time-tooltip-modal.tsx]:
```suggestion
export default OrderTimeTooltipModal;
```

**Review by ameerul-deriv — APPROVED** [review]

**Review by farhan-nurzi-deriv — APPROVED** [review]

File: packages/p2p/src/constants/order-list.js [hunk 4]
```diff
@@ -1,4 +1,36 @@
+import { localize } from 'Components/i18next';
+
 export const order_list = {
     ACTIVE: 'active',
     INACTIVE: 'inactive',
 };
+
+export const order_time_info_message = localize('Orders will expire if they aren’t completed within this time.');
```
Comment (farrah-deriv) [inline packages/p2p/src/constants/order-list.js]: why is this in constants file?

**Review by niloofar-deriv — APPROVED** [review]

**Review by farrah-deriv — APPROVED** [review]

**Pull request merged**